xattr/lib_build.py

```diff
@@ -272,12 +272,12 @@
 #define XATTR_CREATE 0x1
 #define XATTR_REPLACE 0x2
 
-#ifndef u_int32_t
-#define u_int32_t uint32_t
+#ifndef uint32_t
+#define uint32_t uint32_t
```

Comment:
This looks wrong to me; I suspect this whole section can be removed

Reply:
Yes, we wanted to replace this PR with #71. Could you review that one, perhaps? Thanks!